src/app/components/modals/modal-highlight/modal-highlight.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="modal-highlight-container">
+  <button type="button" class="close pull-right" aria-label="Close" (click)="bsModalRef.hide()">
+    <span aria-hidden="true">&times;</span>
+  </button>
+  <div class="modal-body">
+    <h2>Hightlight project</h2>
```

Comment:
```suggestion
    <h2>Highlight project</h2>
```

Reply:
Nice catch, I made this mistake so often in the backend...